src/less/components/form.less

```diff
@@ -198,7 +200,11 @@
  */
 
 .uk-radio:not(:disabled),
-.uk-checkbox:not(:disabled) { cursor: pointer; }
+.uk-radio:not(.uk-disabled),
+.uk-checkbox:not(:disabled),
+.uk-checkbox:not(.uk-disabled) { 
+  cursor: pointer; 
+}
```

Comment:
I just realized that this will not work as either of the selectors will always be true.

Maybe it would be better to split them and reset the cursor to default on `.uk-disabled`. Would be nice to have some input from the UIkit team.